sktime/forecasting/neuralforecast.py

```diff
@@ -168,6 +173,7 @@ class NeuralForecastRNN(_NeuralForecastAdapter):
         # estimator type
         # --------------
         "python_dependencies": ["neuralforecast>=1.6.4"],
+        "capability:global_forecasting": True,
```

Comment:
Maybe keeping it true by default would be more appropriate. Otherwise, codes as below will not work right?
```py
sktime.registry import all_estimators

all_estimators(
    "forecaster", filter_tags={"capability:global_forecasting": True}, as_dataframe=True
)
```